packages/react-dom/src/events/DOMPluginEventSystem.js

```diff
@@ -384,21 +367,6 @@ export function listenToNativeEvent(
   ) {
     target = (rootContainerElement: any).ownerDocument;
   }
-  if (enablePassiveEventIntervention && isPassiveListener === undefined) {
```

Comment:
This moved below since we can now decide it right before attaching the listener.